easybuild/easyconfigs/t/TM-align/TM-align-20180426-intel-2019.01.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'MakeCp'
+
+name = 'TM-align'
+version = '20180426'
+
+homepage = 'https://zhanglab.ccmb.med.umich.edu/RNA-align/TMalign/'
+description = """This package unifies protein structure alignment and RNA structure alignment
+ into the standard TM-align program for single chain structure alignment, MM-align program for
+ multi-chain structure alignment, and TM-score program for sequence dependent structure superposition."""
+
+toolchain = {'name': 'intel', 'version': '2019.01'}
```

Comment:
@pescobar any specific reason why you're not using the `2019a` intel version? we should try to stick with the default toolchains if possible.

Reply:
the only reason is that I have no `2019a` installed in my system but I will update the easyconfig